nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -192,6 +192,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The `aws` package has been removed due to being abandoned by the upstream. It is recommended to use `awscli` or `awscli2` instead.
 
+- The `xdg.portal.gtkUsePortal` option has been removed due to deprecation. Consider not setting the variable or set `GTK_USE_PORTAL`  yourself via `environment.sessionVariables`.
```

Comment:
The option has only been deprecated this summer. It should probably not be removed until stable users see its deprecation. Also it is important to highlight that the option was problematic (e.g. it could make boot several minutes slower).

Reply:
Ah, I'll hold off on that until branch-off then.